csharp/ql/test/resources/stubs/ServiceStack.Text/8.0.0/ServiceStack.Text.cs

```diff
@@ -1042,8 +1042,7 @@ public static partial class ListExtensions
     }
     public class LongRange : System.IEquatable<ServiceStack.LongRange>
     {
-        public virtual ServiceStack.LongRange<Clone>$() => throw null;
```

Comment:
Looks like the stub generator should not emit this.

Reply:
I will look into this problem as well (it is different from the previous ones).

Reply:
This is because the a "clone" method is created for `record class`. It can't be invoked directly (only implicitly via the `r with { ... }` expression, so we should just avoid generating it.

Reply:
Depending on what our goal is, it might not be enough to remove this method from the stubs.

The `with { ... }` expression can only be used with records. So if this class is not generated as a `record class`, then we won't be able to write `new LongRange() with { ... }` in the test files. (But I think we can live with this limitation)